<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QWidget" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员操作</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>201</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="AccountButton">
      <property name="text">
       <string>查询账号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PasswordButton">
      <property name="text">
       <string>修改密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="AuthButton">
      <property name="text">
       <string>修改权限</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>账号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="AccountText"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="PasswordLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>160</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>修改后的密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="PasswordSet"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>账号权限</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="AuthLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
